packages/react-dev-utils/__tests__/getPublicUrlOrPath.test.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const getPublicUrlOrPath = require('../getPublicUrlOrPath');
+
+const tests = [
+  // DEVELOPMENT with homepage
+  { dev: true, homepage: '/', expect: '/' },
+  { dev: true, homepage: '/test', expect: '/test/' },
+  { dev: true, homepage: '/test/', expect: '/test/' },
+  { dev: true, homepage: './', expect: '/' },
```

Comment:
How come this doesn't match production where the expected would also be './'?

Reply:
When starting webpack-dev-server, we have to specify some paths where assets, and js should be served. Having a relative path we can't do that, this is why in dev mode relative paths always will resolve to root `/`

Reply:
Got it. Thanks for the prompt response!